public/assets/images/icons/information-icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26.42 26.42" width="20" height="20">
+    <g id="Layer_2" data-name="Layer 2">
```

Comment:
Remove ids and run the svg through here: https://jakearchibald.github.io/svgomg/